zipline/pipeline/loaders/earnings_estimates.py

```diff
@@ -244,54 +246,40 @@ def get_adjustments(self,
         col_to_overwrites = defaultdict(dict)
         # We no longer need NORMALIZED_QUARTERS in the index, but we do need it
         # as a column to calculate adjustments.
-        zero_qtr_data = zero_qtr_data.reset_index(NORMALIZED_QUARTERS)
-
-        for sid_idx, sid in enumerate(assets):
-            zero_qtr_sid_data = zero_qtr_data[
-                zero_qtr_data.index.get_level_values(SID_FIELD_NAME) == sid
-            ]
-            # Determine where quarters are changing for this sid.
-            qtr_shifts = zero_qtr_sid_data[
-                zero_qtr_sid_data[NORMALIZED_QUARTERS] !=
-                zero_qtr_sid_data[NORMALIZED_QUARTERS].shift(1)
-            ]
-            # On dates where we don't have any information about quarters,
-            # we will get nulls, and each of these will be interpreted as
-            # quarter shifts. We need to remove these here.
-            qtr_shifts = qtr_shifts[
-                qtr_shifts[NORMALIZED_QUARTERS].notnull()
+        zero_qtr_data = zero_qtr_data.reset_index(level=NORMALIZED_QUARTERS)
+        zero_qtr_data.sort_index(inplace=True)
```

Comment:
Ah, and we had it sorted by event_date before, so that would have screwed up the order of the days